src/app/containers/Image/helpers/srcSetHelper.js

```diff
@@ -0,0 +1,27 @@
+const createSrcset = (originCode, locator, originalImageWidth) => {
+  const resolutions = [240, 320, 480, 624, 800];
+  const ichefUrl = 'https://ichef.bbci.co.uk/news';
+  if (originCode === 'pips') {
+    return null;
+  }
+
+  const requiredResolutions = resolutions.filter(
+    resolution => resolution <= originalImageWidth,
+  );
+
+  if (
+    originalImageWidth < resolutions[resolutions.length - 1] &&
+    !requiredResolutions.includes(originalImageWidth)
+  ) {
+    requiredResolutions.push(originalImageWidth);
+  }
+
+  const urls = requiredResolutions.map(
```

Comment:
I feel like we should be able to reuse or refactor this method below in some way... 

https://github.com/bbc/simorgh/blob/3f152e49d3f0d3da4a7fa28e6cfaf9d1427cbc04/src/app/containers/Image/index.jsx#L24-L29

Reply:
Me too, but I'm not sure of the best way.

Like a function that just returns the string?

Reply:
Yeah, we should only construct ichef urls in 1 place, you can bring the getIChefURL function into its own file and use it in both places